queries/tact/textobjects.scm

```diff
@@ -0,0 +1,278 @@
+    "{"
+    (_) @_start @_end
+    (_)? @_end
+    "}")
+  (#make-range! "loop.inner" @_start @_end)) @loop.outer
+
+(repeat_statement
+  body: (block_statement
+    "{"
+    (_) @_start @_end
+    (_)? @_end
+    "}")
+  (#make-range! "loop.inner" @_start @_end)) @loop.outer
+
+(do_until_statement
+  body: (block_statement
+    "{"
+    (_) @_start @_end
+    (_)? @_end
+    "}")
+  (#make-range! "loop.inner" @_start @_end)) @loop.outer
+
+(foreach_statement
+  body: (block_statement
+    "{"
+    (_) @_start @_end
+    (_)? @_end
+    "}")
+  (#make-range! "loop.inner" @_start @_end)) @loop.outer
+
+; conditional.inner & outer
+; -------------------------
+(if_statement
+  consequence: (block_statement
+    "{"
+    (_) @_start @_end
+    (_)? @_end
+    "}")
+  (#make-range! "conditional.inner" @_start @_end)) @conditional.outer
+
+(if_statement
+  alternative: (else_clause
+    (block_statement
+      "{"
+      (_) @_start @_end
+      (_)? @_end
+      "}"))
+  (#make-range! "conditional.inner" @_start @_end)) @conditional.outer
+
+(if_statement) @conditional.outer
+
+; block.inner & outer
+; -------------------
+(_
+  (block_statement) @block.inner) @block.outer
+
+; call.inner & outer
+; ------------------
+(method_call_expression
+  arguments: (argument_list
+    "("
+    (_) @_start
+    (_)? @_end
+    ")")
+  (#make-range! "call.inner" @_start @_end)) @call.outer
+
+(static_call_expression
+  arguments: (argument_list
+    "("
+    (_) @_start
+    (_)? @_end
+    ")")
+  (#make-range! "call.inner" @_start @_end)) @call.outer
+
+(instance_expression
+  arguments: (instance_argument_list
+    "{"
+    (_) @_start
+    (_)? @_end
+    "}")
+  (#make-range! "call.inner" @_start @_end)) @call.outer
+
+((initOf
+  name: (identifier) @_name
+  arguments: (argument_list
+    "("
+    (_) @_start
+    (_)? @_end
+    ")") @_args)
+  (#make-range! "call.outer" @_name @_args)
+  (#make-range! "call.inner" @_start @_end))
+
+; return.inner & outer
+; --------------------
+(return_statement
+  (_) @return.inner) @return.outer
+
+; number.inner
+; ------------
+(integer) @number.inner
+
+; parameter.inner & outer
+; -----------------------
+(parameter_list
+  ((_) @parameter.inner
+    .
+    ","? @parameter.outer) @parameter.outer)
+
+(argument_list
+  ((_) @parameter.inner
+    .
+    ","? @parameter.outer) @parameter.outer)
+
+(instance_argument_list
+  ((_) @parameter.inner
+    .
+    ","? @parameter.outer) @parameter.outer)
+
+; assignment.inner & outer w/ lhs & rhs
+; -------------------------------------
+(let_statement
+  name: (_) @assignment.lhs
+  value: (_) @assignment.inner @assignment.rhs) @assignment.outer
+
+(constant
+  name: (_) @assignment.lhs
+  value: (_) @assignment.inner @assignment.rhs) @assignment.outer
+
+(field
+  name: (_) @assignment.lhs
+  value: (_) @assignment.inner @assignment.rhs) @assignment.outer
+
+; comment.inner & outer
+; ---------------------
+(comment) @comment.inner
+
+(comment)+ @comment.outer
```

Comment:
Unfortunately quantified captures aren't supported here yet (will change soon after Neovim 0.10) so maybe best to comment this query out for now and just tack on `@comment.outer` to the inner query too